### ISSUE TITLE
Karazhan bugs in raid (Opera Event, NightBane and The curator)

### ISSUE THREAD

**Issue by auroralima** [talamortis/OregonCore#84]
**Operating system:** Ubuntu 18

Current behavior: we found some bugs in karazhan, that's pretty annoying because it's the only raid we release lol since we're doing progressive
Well the first one we found was at the Opera Event, the first time we went, we took the wizard of oz ... two problems happened there ...
1 - the loss there is bugged, ranged and healer can only use skills if they practically stack
2 - we wiped, when we became angels and tried to enter the raid, said that could not enter the fight and all were out, it took the vhiper to come and take the locked, redo the raid to enter ...

another mistake we discovered upon arriving at the nightbane room https://tbcdb.com/?npc=17225
he was not there, I went there and added him, and he came to be a target, went flying flying until he disappeared lol.
we left him there, since there was nothing to be done, and we went up to the curator ...
we started the fight in a good way, it was working properly as it should be, until suddenly ...

SS's
https://prnt.sc/q0vqj1 - The Curator
https://prnt.sc/q0vqrn - Nightbane

**Comment by talamortis** [comment]
- [x] The Curator - Done https://github.com/talamortis/OregonCore/commit/ce8976737c6fcc74e227242bbc821c42057fff6c
- [x] Opera - Done https://github.com/talamortis/OregonCore/commit/eafc89b6182ac68ded4b4ba313731fe335528e8f
- [ ] Nightbane

**Comment by auroralima** [comment]
<3 Good job man

**Comment by coolzoom** [comment]
Nightbane needs to be summoned use an item, did do that?

**Comment by talamortis** [comment]
Yes, the issue is the script mate, no point working on it till the restructure is complete :)

- Pathing of nightbane is horrible. 
- also evades when waypoint reached.
